test/test_bitarray.rb

```diff
@@ -1,5 +1,5 @@
 require "minitest/autorun"
-require "bitarray"
+require_relative "../lib/bitarray"
```

Comment:
This allows running the tests from the root directory or from the test directory.